ibis/expr/types/relations.py

```diff
@@ -1881,8 +1882,8 @@ def difference(self, table: Table, *rest: Table, distinct: bool = True) -> Table
         └───────┘
         """
         node = ops.Difference(self, table, distinct=distinct)
-        for table in rest:
-            node = ops.Difference(node, table, distinct=distinct)
+        for other in rest:
+            node = ops.Difference(node, other, distinct=distinct)
```

Comment:
Can we leave these as they were just to reduce review noise?

Reply:
pre-commit's ruff insisted. Happy to remove.